src/content/docs/service-level-management/intro-slm.mdx

```diff
@@ -10,15 +10,22 @@ metaDescription: How to create and consume SLIs and SLOs with service level mana
 
 import slmServiceLevelsOverview from 'images/slm_screenshot-full_service-levels-overview.png'
 
-With New Relic you can define and consume service level indicators and service level objectives for your applications.
+Service levels were popularized by Google and became a best practice in the industry. Service levels complement existing monitoring by adding a perspective more focused on the business and the user experience.
+
+With New Relic, you can define and consume service level indicators (SLIs) and service level objectives (SLOs) for your applications. There are several relevant reasons to configure and follow your New Relic service levels:
+
+- Easy creation: facilitates creation of service levels with different modes of complexity, from one-click to advanced modes fully customizable for advanced users. 
+- Integrated with New Relic experiences: visualize and work with services levels in Navigator, Workloads, mini overview and through most of the New Relic observability tools.
+- Alerts support: create alerts that warn you of any degradation that you should pay attention to.
+- Analysis tooling: analyze the reliability status of all systems in a consistent way. The Period over Period view mode allows you to spot trend changes in any service level. Additionally, whenever a Service Level is breached, the summary view allows you to check potential causes of the issue. 
 
 ## What are SLIs and SLOs? [#what-sli-slo]
 
 Service levels are used to measure the performance of a service from the end user (or client application) point of view. For instance, a service level can represent whether a video loaded quickly enough, or whether a directions service returned at least one possible route between two points.
 
-**Service level indicators (SLIs) are accurate quantitative measures of the user experience** as described by a service level. They represent a proportion of successful outputs, and therefore they’re expressed as a percentage (%). For example, an SLI can measure the proportion of requests that were faster than some threshold, or the proportion of records coming into a pipeline that resulted in the correct value coming out.
+**Service level indicators are accurate quantitative measures of the user experience** as described by a service level. They represent a proportion of successful outputs, and therefore they’re expressed as a percentage (%). For example, an SLI can measure the proportion of requests that were faster than some threshold, or the proportion of records coming into a pipeline that resulted in the correct value coming out.
```

Comment:
why did we take out the acronyms when we mention the acronyms later?

Reply:
The doc uses the acronym even before this, so we define it even earlier.